x-pack/legacy/plugins/reporting/export_types/csv/server/lib/__tests__/hit_iterator.ts

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import sinon from 'sinon';
+import { CancellationToken } from '../../../../../common/cancellation_token';
+import { Logger, ScrollConfig } from '../../../../../types';
+import { createHitIterator } from '../hit_iterator';
+
+const mockLogger = {
+  error: new Function(),
+  debug: new Function(),
+  warning: new Function(),
+} as Logger;
+const debugLogStub = sinon.stub(mockLogger, 'debug');
+const warnLogStub = sinon.stub(mockLogger, 'warning');
+const errorLogStub = sinon.stub(mockLogger, 'error');
+const mockCallEndpoint = sinon.stub();
```

Comment:
optional: Any reason not to convert to `jest.fn()` instead of `sinon` here?

Reply:
Well, I added this as a mocha test since there was already a `__tests__` server. Hope that's ok.